src/azure-cli/azure/cli/command_modules/configure/commands.py

```diff
@@ -18,3 +18,7 @@ def load_command_table(self, _):
         g.command('show', 'show_cache_contents')
         g.command('delete', 'delete_cache_contents')
         g.command('purge', 'purge_cache_contents')
+
+    with self.command_group('local-context', configure_custom, is_experimental=True) as g:
+        g.command('on', 'turn_local_context_on')
+        g.command('off', 'turn_local_context_off')
```

Comment:
The command name comes from PM? According to command name convention, the last word should be a verb. I'm personally fine with the on/off, not sure if we have simliar existing commands.

Reply:
yes, PM prefer such commands ...